package.json

```
@@ -3,7 +3,7 @@
   "type": "module",
   "version": "0.0.1",
   "scripts": {
-    "dev": "astro dev",
+    "dev": "astro dev --host",
```

Debo pasarle algun parametro al ejecutar `npm run dev` ?

---

No requiere de parámetros

---

Para que sirve --host ?

---

Para brindar una url local y poder acceder desde otro dispositivo como el celular

---

Para estos casos tan especificos creo que es mejor crear otro atributo para no usar el "dev"

```suggestion
    "dev": "astro dev",
    "dev-host": "astro dev --host",
```

---

Hice el cambio, pero considero que por ser un caso que no se usara frecuentemente debería ser el cambio ahí mismo